python/mxnet/ndarray/ndarray.py

```diff
@@ -46,7 +46,7 @@
            "ones", "add", "arange", "eye", "divide", "equal", "full", "greater", "greater_equal",
            "imdecode", "lesser", "lesser_equal", "logical_and", "logical_or", "logical_xor",
            "maximum", "minimum", "moveaxis", "modulo", "multiply", "not_equal", "onehot_encode",
-           "power", "subtract", "true_divide", "waitall", "_new_empty_handle", "histogram",
+           "pow", "power", "subtract", "true_divide", "waitall", "_new_empty_handle", "histogram",
```

Comment:
I do not think it's a good practice to intentionally introduce multiple APIs that do the same thing. Since there is `numpy.power`, I'd rather add `symbol.power`, document `symbol.pow` is deprecated and will be replaced by sym.power.

Reply:
Okay